src/command-line/install.js

```diff
@@ -38,45 +38,54 @@ program
 			const packagesParent = path.dirname(packagesPath);
 			const packagesConfig = path.join(packagesParent, "package.json");
 
-			// Create node_modules folder, otherwise npm will start walking upwards to find one
+			// Create node_modules folder, otherwise yarn will start walking upwards
+			// to find one
 			fsextra.ensureDirSync(packagesPath);
 
-			// Create package.json with private set to true to avoid npm warnings, if
-			// it doesn't exist already
+			// Create package.json with private set to true, if it doesn't exist
+			// already
 			if (!fs.existsSync(packagesConfig)) {
 				fs.writeFileSync(packagesConfig, JSON.stringify({
 					private: true,
 					description: "Packages for The Lounge. All packages in node_modules directory will be automatically loaded.",
 				}, null, "\t"));
 			}
 
-			const npm = child.spawn(
-				process.platform === "win32" ? "npm.cmd" : "npm",
+			let success = false;
+			const add = child.spawn(
+				"./node_modules/.bin/yarn",
```

Comment:
@xPaw, could you test on Windows with my latest commit?
I am using `./node_modules/.bin/yarn`, which is actually the same as `./node_modules/yarn/bin/yarn.js` and **not** `./node_modules/yarn/bin/yarn`. The `yarn.cmd` one simply calls `yarn.js` (`yarn/bin/yarn` and `yarn/bin/yarn.cmd` are useful to open new terminal on double-click, if I understand correctly) so `.bin/yarn` should actually be cross-platform.

Reply:
It's not classform and still needs needs the `.cmd`. Either that or directly run `node yarn.js`, should be able to get current node binary to run under it.

Reply:
I switched to using `process.execPath` and `yarn.js` to avoid any ambiguity.